frontend/src/charts/TableChart.tsx

```diff
@@ -186,7 +197,8 @@ export function TableChart(props: TableChartProps) {
       ) : (
         <figure>
           <figcaption style={titleStyle}>
-            Data breakdown summary for {sentenceCaseName} in the United States
+            Data breakdown summary for {sentenceCaseName} in{' '}
```

Comment:
do you think we need data here? Maybe just `Breakdown summary for ...` works fine? Any chance we get to shorten our text and labels I think we should take it